collector.spec

```diff
@@ -0,0 +1,54 @@
+%global goipath         github.com/os-observability/redhat-opentelemetry-collector
+
+Version:                0.92.0
+ExcludeArch:            %{ix86} s390 ppc ppc64 aarch64
+
+%gometa
+
+%global common_description %{expand:
+TODO}
+
+%global golicenses    LICENSE
+%global godocs        README.md
+
+Name:           redhat-opentelemetry-collector
```

Comment:
Can we use the name `defined` in the Makefile here?

Reply:
Same as above.